CMakeLists.txt

```diff
@@ -24,7 +24,7 @@
 
 cmake_minimum_required(VERSION 3.1)
 # XXX using 0.1.90 instead of 0.1.0-DEV
-project(nghttp2 VERSION 0.1.90)
+project(ngtcp2 VERSION 0.1.90)
```

Comment:
This fixes the name of the Visual Studio solution when using `cmake -G "Visual Studio ..."`.